### PR TITLE
tests/vm_test: code cleanup

### PR DESCRIPTION
**Pull request by ShellyKa13**
Remove unused variables.
Check unchecked errors.
Prevent from failing Eventually incase of possible temp issue with client availability.

-->
```release-note
NONE
```

### CODE REVIEW

File: tests/vm_test.go [hunk 1]
```diff
@@ -451,20 +447,20 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					)).GeneratePayload()
 				Expect(err).ToNot(HaveOccurred())
 
-				vm, err = virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, resourcesPatch, metav1.PatchOptions{})
+				_, err = virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, resourcesPatch, metav1.PatchOptions{})
```
Comment (codingben) [inline tests/vm_test.go:450]: Can you help me to understand why this patch is needed, by updating `/spec/template/spec/domain/resources` after stopping the VM?

**Comment by ShellyKa13** [comment]
/kind cleanup
@codingben they changed it to kind cleanup :)

File: tests/vm_test.go (hunk 1 above)
Comment (ShellyKa13) [reply tests/vm_test.go:450]: not sure I understand your question..
If is it why change the vm and not just restart then I think its in order to compare the revision and see that it changed

Comment (codingben) [reply tests/vm_test.go:450]: Got it, thanks :)

**Review by codingben — APPROVED** [review]
/lgtm

**Review by enp0s3 — COMMENTED** [review]
/approve
@ShellyKa13 Thank you

**Pull request merged**